detroit-ui-components/src/overlays/Modal.tsx

```diff
@@ -21,12 +21,19 @@ export interface ModalProps extends Omit<OverlayProps, "children"> {
 }
 
 const ModalHeader = (props: { title: string; uniqueId?: string; className?: string }) => {
+  const modalHeader = useRef<HTMLHeadingElement>(null)
+  useEffect(() => {
+    if (modalHeader.current && props.title) {
+      modalHeader?.current?.focus()
+    }
+  }, [props.title])
+
   const classNames = ["modal__title"]
   if (props.className) classNames.push(props.className)
   return (
     <>
       <header className="modal__header">
-        <h1 className={classNames.join(" ")} id={props.uniqueId}>
+        <h1 ref={modalHeader} tabIndex={-1} className={classNames.join(" ")}>
```

Comment:
Hesitant to remove this id prop... don't see it used in Detroit but imagine it could be used by consumers since it exists in the UI-C version as well

Reply:
It looks like it is used internally in this component - the encompassing overlay gets "ariaLabelledBy=uniqueId" so that the overlay points to the header content - so I would keep the id here :)

Reply:
Ahh I see now! Thank you!

Reply:
oh yes, my bad 😞 Were testing something, and accidentally removed it 😅